cogeo_mosaic/scripts/cli.py

```diff
@@ -77,7 +77,7 @@ def create(
     quiet,
 ):
     """Create mosaic definition file."""
-    input_files = input_files.read().splitlines()
+    input_files = input_files.read().strip().splitlines()
```

Comment:
```suggestion
    input_files = [file.strip() for file in input_files if file.strip()]
```

Reply:
as @kylebarron noted (via slack) the file is opened in read mode so we don't need the `splitline` and can iterate over the lines.

We also have a preference for list comprehension